### ISSUE TITLE
Missing swig files?

### ISSUE THREAD

**Issue by stucchio** [fmder/ghalton#6]
Greetings,

I'm trying to build a faster version of ghalton - one that will generate a numpy array directly, rather than building an intermediate python list. (That part is kind of slow compared to everything else). 

Unfortunately, unless I made a mistake, the repo seems to be missing the swig interface file. Any chance that could be added?

**Comment by fmder** [comment]
You are right, but I can't find the file anymore (It was on my lab computer
back when I was at school) ... However, I think guess it was not that
complex: the api is quite simple. If you happen to create a new one please
send a PR.

Cheers,

2015-12-16 10:28 GMT-05:00 C. Stucchio <notifications@github.com>:

**Comment by SynBioFoundry** [comment]
This worked for me, but is incomplete because it does not provide access to `ghalton.EA_PERMS` and `ghalton.PRIMES`, which may not be needed anyway:

```
/* file : Halton.i */
%module ghalton
%{
#include "Halton.hpp"
%}

/* %include "Constants.hpp" */
%include "Halton.hpp"
```

Compile with:

`swig -Wall -c++ -python Halton.i`

**Comment by fmder** [comment]
I may have added them manually to the ghalton.py file. I should reorganize things to make this automatically.

**Comment by fmder** [comment]
Ok, thisi is fix in 0.6.1

**Comment by fmder** [comment]
@stucchio I'm interested by your modification to generate numpy arrays directly please PR if you realized your project.
